README.md

```diff
@@ -375,63 +372,58 @@ EBS volume needs to be created as well. See [gitlab.tf.json.template.py] and the
-
-## 3.3 Deploying lambda functions
-
-Once the cloud infrastructure for your deployment has been provisioned, you can
-deploy the project code into AWS Lambda. Running
+to prepare the lambda functions defined in the `lambdas` directory for 
+deployment via Terraform. Then run
 
 ```
-make deploy
+make terraform
 ```
 
-Will create or update AWS Lambda functions for each lambda defined in the
-`lambdas` directory. It will also create or update an AWS API Gateway to proxy
-the functions that act as web services. We call those functions *API lambdas*.
+which will display a plan and ask you to confirm it. Please consult the 
+Terraform documentation for details. You will need to run `make terraform` once 
+to set up your deployment and again to deploy any code changes.
 
-## 3.4 Provisioning stable API domain names
+Any time you wish to change the code running in the lambdas you will need to 
+run `make deploy` followed by `make terraform`. 
+
+Some Terraform configuration is generated by `make deploy`, but the rest is
+defined in `….tf.json` files which in turn are generated from 
+`….tf.json.template.py` files which are simple Python scripts containing the
+desired JSON as Python dictionary and list literals and comprehensions. Changes
+to these files require running `make terraform` again in order to update
+cloud infrastructure for the selected deployment.
+
+## 3.3 Locating deployment endpoints via DNS
 
 The HTTP endpoint offered by API Gateway have somewhat cryptic and hard to
 remember domain names:
 
-https://klm8yi31z7.execute-api.us-east-1.amazonaws.com/hannes/
+> https://klm8yi31z7.execute-api.us-east-1.amazonaws.com/hannes/
 
 Furthermore, the API ID at the beginning of the above URL is likely to change
-when you accidentally delete the REST API and then recreate it. To provide
-stable and user-friendly URLs for the API lambdas, we provision a *custom domain
-name* object in API Gateway along with an ACM certificate and a CNAME record in
-Route 53. Running `make terraform` again after `make deploy` will detect the
-newly deployed API lambdas and create those resources for you. What the
-user-friendly domain names look like depends on project configuration. The
-default for HCA is currently
+any time the REST API is re-provisioned.  To provide stable and user-friendly 
+URLs for the API lambdas, we provision a *custom domain name* object in API 
+Gateway along with an ACM certificate and a CNAME record in Route 53. the 
+user-friendly domain names depends on project configuration. The default for 
+HCA is currently
 
-http://indexer.${AZUL_DEPLOYMENT_STAGE}.singlecell.gi.ucsc.edu/
-http://service.${AZUL_DEPLOYMENT_STAGE}.singlecell.gi.ucsc.edu/
+> http://indexer.${AZUL_DEPLOYMENT_STAGE}.singlecell.gi.ucsc.edu/
+> http://service.${AZUL_DEPLOYMENT_STAGE}.singlecell.gi.ucsc.edu/
 
 Personal deployments are subdomains of the domain for the ``dev`` deployment:
 
 http://indexer.${AZUL_DEPLOYMENT_STAGE}.dev.singlecell.gi.ucsc.edu/
```

Comment:
Here the quote (`>`) markup is missing. This should be consistent with the sections above.